README.md

```diff
@@ -1,50 +1,72 @@
-# RSpec Buildkite Analytics
+# Buildkite Collectors for Ruby
+
+Official Ruby-based [Buildkite Test Analytics](https://buildkite.com/test-analytics) collectors ✨
+
+
+**Supported test frameworks:** RSpec, Minitest and more [coming soon](#roadmap).
+
+**Supported CI systems:** Buildkite, GitHub Actions, CircleCI, Jenkins, and others via the `BUILDKITE_ANALYTICS_*` environment variables.
 
 This gem collects data about your test suite's performance and reliability, and allows you to see trends and insights about your test suite over time ✨
 
-## Installation
+## Installing
 
-1. Create a new branch
+### RSpec
```

Comment:
Do we have instructions for minitest?

Reply:
Not yet. I think it would not change much!